config/arpwatch.xml

```diff
@@ -81,9 +81,12 @@
 	</additional_files_needed>
 	<fields>
 		<field>
-			<fielddescr>Listening Interface</fielddescr>
-			<fieldname>interface</fieldname>
-			<description>Choose the desired listening interface here.</description>
+			<fielddescr>Interfaces</fielddescr>
+			<fieldname>interface_array</fieldname>
```

Comment:
It will, you will have to select the interfaces you want to monitor after its installed.
It will look like this now:
		<arpwatch>
			<config>
				<interface_array>lan</interface_array>
				<interface_array>opt1</interface_array>
				<interface_array>opt2</interface_array>
				<enable_email>on</enable_email>
			</config>
		</arpwatch>